subsys/nonsecure/Kconfig

```
@@ -0,0 +1,262 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if BUILD_WITH_TFM || IS_SPM
+menu "Trusted Execution Non-Secure"
+
+menu "Peripheral Secure mapping"
+
+config NRF_FPU_SECURE
+	bool "FPU configured as secure peripheral"
+	depends on CPU_HAS_FPU
+	help
+	  FPU is implemented with user-selectable security mapping.
+          If it has been configured as a secure peripheral then only the secure
+          processing environment is able to access the peripheral
+
+peripheral=NRF_DCNF_SECURE
+rsource "Kconfig.template.peripheral_secure"
```

I guess this was supposed to be:
```suggestion
peripheral=DCNF
rsource "Kconfig.template.peripheral_secure"
```
And similarly for a few next entries.